ERCS/erc-7001.md

```diff
@@ -0,0 +1,86 @@
+---
+eip: 7001
```

Comment:
```suggestion
eip: 7624
```
Assigning next sequential EIP/ERC/RIP number.
Numbering changed to sequential from 7500.

Please also update the filename.